packages/api/tests/lambdas/sf-event-sqs-to-db-records/test-index.js

```diff
@@ -53,9 +56,18 @@ const {
   '@cumulus/aws-client/StepFunctions': {
     describeExecution: () => Promise.resolve({}),
   },
+  '@cumulus/message/Executions': {
```

Comment:
Can we avoid increasing the use of proxyquire here by passing an optional event param to the handler method?  

e.g. 

`const getCumulusMessageFromExecutionEvent = event.coreTestMethods.getCumulusMessageFromExecutionEvent || getCumulusMessageFromExecutionEvent`

or the like?

Reply:
I looked into this, but persisting that function all the way down the call tree to the relevant place gives me bad feels.

Reply:
Hmm, yeah, I see, because it's an SQS event you'd wind up encoding the method or something like that, which is...admittedly not great either.   Fair enough.